sw/simulation/agents/particle_oriented/particle_oriented.cpp

```diff
@@ -28,10 +28,10 @@ vector<float> particle_oriented::state_update(vector<float> state)
     vy_des = manualy;
     dpsirate = manualpsi_delta;
   }
-  controller->saturate(vx_des);
-  controller->saturate(vy_des);
+  // controller->saturate(vx_des);
```

Comment:
I'm writing here but it goes for the whole file. Please make a new agent if this one doesn't fit the purpose (no acceleration I see?) Or else the other controllers that use it may not work anymore.

In the readme file of your controller, write which agent the controller works with.
In the readme file of your agent, write which controller(s) the agent can work with.